### PR TITLE
 Solved ValueError Issues in Placement cell module

### PR DESCRIPTION
**Pull request by geetikn**
Solved the Value error as mentioned in Issue #516 , #536 and #537

### CODE REVIEW

**Review by chiragagrawal026 — COMMENTED** [review]
This can not be considered a fix of the mentioned issues. You need to make the page work, not just remove the error it is getting and render a blank page.

File: FusionIIIT/applications/placement_cell/views.py [hunk 1]
```diff
@@ -159,11 +160,12 @@ def get_reference_list(request):
         context = {
             'reference_objs': reference_objects
         }
-        return JsonResponse(context)
+    return HttpResponse(context)
```
Comment (chiragagrawal026) [inline FusionIIIT/applications/placement_cell/views.py]: You have just defined an empty context object above which is rendered finally as a blank page, and the if condition is not even executing.

File: FusionIIIT/applications/placement_cell/views.py [hunk 2]
```diff
@@ -175,18 +177,20 @@ def company_name_dropdown(request):
             'company_names': company_name
         }
 
-        return JsonResponse(context)
+    return HttpResponse(context)
```
Comment (chiragagrawal026) [inline FusionIIIT/applications/placement_cell/views.py]: The same goes for this and '/checking_roles' url.

**Comment by geetikn** [comment]
Solved Issue #620 and #621.

**Comment by MananJethwani** [comment]
@chiragagrawal026 kindly review this PR again

**Comment by chiragagrawal026** [comment]
@MananJethwani  according to me, this pull request should be closed as there is a separate PR addressing these two mentioned issues 620 & 621. Also, this PR was earlier created for fixing issues 516, 536 and 537, but they cannot be solved by a single PR. Those issues will be needing their separate code changes.

**Comment by MananJethwani** [comment]
ok closing this PR @geetikn kindly open 3 separate PRs

**Pull request closed without merging**